src/utils/editor/index.js

```diff
@@ -99,14 +99,12 @@ export function toSourceLocation(
   location: EditorPosition
 ): AstPosition {
   return {
-    line: toSourceLine(sourceId, location.line),
+    line: toSourceLine(sourceId, location.line || 0),
```

Comment:
whats the rationale with `|| 0` again?

Reply:
It will be really hard to get rid of this pesky `|| 0`. I introduced `EditorPosition` when adding wasm capability, so intent was to not have `line: ?number` there. Can we investigate this and make sure wasm related utilites will not have possible void return?

Reply:
@yurydelendik what would it take to investigate?

can you outline the STR w/ fib?
http://firefox-dev.tools/debugger-examples/examples/wasm/fib/fib.index.html

do you need to poke around w/ this PR?

Reply:
@yurydelendik the only problem was that you must have missed the Flow header in the file where you defined `EditorPosition` because when I added it the errors appeared. 🙁 I could revert the changes including the header to avoid this here. But I think it is problematic that we have void returns.